django_ecommerce/settings.py

```diff
@@ -20,14 +21,15 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-g%2@m997h_a#=jtwvse5yrmae@3mb#5=fak-m93!=jmf6l$()k'
+SECRET_KEY = 'django-insecure-b=+&bq(#sm9srpii+13qo!7uq!ixxtg5s1h^m^o79efd54fqk0'
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
Same as above. 
If this is stored in `.env`, we will have better flexibility for local and production. As on production, `DEBUG` must always be set to `False` as shown in the security warning message above.

Maybe we can do something like this: 

- Create `.env` file in `django_ecommerce`
- Put `.env` in gitignore (very important)
- Also create `.env.example` file with the env keys only (not the value) so other developers can get to know what env variables are being used inside the codebase.
- In `.env` file: set a variable `DEBUG_MODE=TRUE/FALSE`
- In `settings.py`: set `DEBUG = os.getenv("DEBUG_MODE") == "TRUE" `